app/views/pages/cookies.html.haml

```diff
@@ -0,0 +1,69 @@
+- content_for :page_title do
+  Cookies
+
+.grid-row
+
+  .column-two-thirds
+    %h1.heading-xlarge Cookies
+
+    %p.lede ‘Teaching Jobs’ puts small files (known as ‘cookies’) onto your computer to collect information about how you use the site.
+
+    %p Cookies are used to:
+
+    %ul.list.list-bullet
+      %li measure how you use the this service so it can be updated and improved based on your needs
+      %li remember the notifications you’ve seen so that we don’t show them to you again
+      %li remember who you are when you have logged in so we can show you your information and information relevant to you
+
+    %p ‘Teaching Jobs’ cookies aren’t used to identify you personally.
+    %p Find out more about #{link_to 'how to manage cookies.', 'http://www.aboutcookies.org/'}
+
+    %h2.heading-large How we use cookies
+
+    %h3.heading-medium Measuring website usage (Google Analytics)
+
+    %p We use Google Analytics software to collect information about how you use ‘Teaching Jobs’. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.
+
+    %p Google Analytics stores information about:
+
+    %ul.list.list-bullet
+      %li the pages you visit on ‘Teaching Jobs’
+      %li how long you spend on each page
+      %li how you got to the site
+      %li what you click on while you’re visiting the site
+
+    %p We don’t allow Google to use or share our analytics data.
+
+    %p Google Analytics sets the following cookies:
+
+    %table.mb1
```

Comment:
The cookies below are set by our application, not GA. 

The `seen_cookie_message` it set by the `gov_uk_template` and the definition in the [GOV UK cookie policy](https://www.gov.uk/help/cookies#our-introductory-message)  is

> Saves a message to let us know that you’ve seen our cookie message, expiry 1 month